### PR TITLE
Rlarsson/replymessage whatsapp optin methods

### PR DESCRIPTION
**Pull request by robin-larsson**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/SmartDialog/apiDefinition.swagger.json [hunk 1]
```diff
@@ -1,808 +1,1376 @@
...
-              "type": "object",
-              "properties": {
-                "customerUsers": {
-                  "type": "array",
-                  "items": {
-                    "type": "object",
-                    "properties": {
-                      "customerId": {
+    "swagger": "2.0",
+    "info": {
```
Comment (sriyen-msft) [inline certified-connectors/SmartDialog/apiDefinition.swagger.json:3]: @robin-larsson Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

File: certified-connectors/SmartDialog/apiDefinition.swagger.json [hunk 2]
```diff
@@ -1,808 +1,1376 @@
...
-                      "Customer-Id": {
-                        "parameter": "Customer-Id"
-                      }
+        },
+        "/whatsapp/opt-in/sender/{Sender}/recipient/{Recipient}": {
+            "delete": {
+                "responses": {
+                    "204": {
+                        "description": "default",
+                        "schema": {}
```
Comment (sriyen-msft) [inline certified-connectors/SmartDialog/apiDefinition.swagger.json:1278]: @robin-larsson Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

Applies the same comment for all the empty schemas in the swagger

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by robin-larsson** [comment]
@sriyen-msft the requested changes have now been made.

**Comment by natalie-pienkowska** [comment]
Hi @robin-larsson! Our certification engineer, Srikanth, is currently on emergency leave. Please expect a delay with your PR. He will return January 31st, 2022. We apologize for any inconveniences.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**